[FR] Modding stubs (#5189)
ppy/osu-wiki: wiki/Modding/Nuke/fr.md

---
stub: true
---

# Nuke

> Les **Nukes** étaient un outil de modération des [beatmaps](/wiki/Beatmap) dans les premières années d'osu!. Toute beatmap qui était considéré comme ne respectant pas les règles (ou dont le créateur ne respectait pas le [Code de conduite](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)) pouvait être "nuked" par le [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team) ou la [Global Moderation Team (GMT)](/wiki/People/The_Team/Global_Moderation_Team).

Reviewer: ```suggestion
Les **Nukes** étaient un outil de modération des [beatmaps](/wiki/Beatmap) dans les premières années d'osu!. Toute beatmap qui était considérée comme ne respectant pas les règles (ou dont le créateur ne respectait pas le [Code de conduite](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)) pouvait être "nuked" par le [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team) ou la [Global Moderation Team (GMT)](/wiki/People/The_Team/Global_Moderation_Team).
```

> Les **Nukes** étaient un outil de modération des [beatmaps](/wiki/Beatmap) dans les premières années d'osu!. Toute beatmap qui était considérée comme ne respectant pas les règles (ou dont le créateur ne respectait pas le [Code de conduite](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)) pouvait être "nuked" par le [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team) ou la [Global Moderation Team (GMT)](/wiki/People/The_Team/Global_Moderation_Team).

Reviewer: ```suggestion
Les **Nukes** étaient un outil de modération des [beatmaps](/wiki/Beatmap) dans les premières années d'osu!. Toute beatmap qui était considérée comme ne respectant pas les règles (ou dont le créateur ne respectait pas le [Code de conduite](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping)) pouvait être "nuked" par la [Beatmap Appreciation Team (BAT)](/wiki/Modding/Beatmap_Appreciation_Team) ou la [Global Moderation Team (GMT)](/wiki/People/The_Team/Global_Moderation_Team).
```

> Lorsqu'une beatmap était "nuked", il avait un symbole radioactif ![](/wiki/shared/icon/nuke.gif) sur son fil de discussion. Cela signalait aux autres membres du BAT/GMT qu'il ne devait pas être considéré comme [Ranked](/wiki/Beatmap_ranking_procedure#ranked) tant que des changements significatifs n'avaient pas été effectués. Si une beatmap a effectué les modifications correspondantes, l'icône sera supprimée et il pourra poursuivre le [processus de classement](/wiki/Beatmap_ranking_procedure).

Reviewer: ```suggestion
Lorsqu'une beatmap était "nuked", il avait un symbole radioactif ![](/wiki/shared/icon/nuke.gif) sur son fil de discussion. Cela signalait aux autres membres du BAT/GMT qu'elle ne devait pas être considérée pour la section [Classée](/wiki/Beatmap_ranking_procedure#ranked) sans changements importants. Si une beatmap a effectué les modifications correspondantes, l'icône sera supprimée et elle pourra poursuivre le [processus de classement](/wiki/Beatmap_ranking_procedure).
```

Les beatmaps peuvent être nuked pour une multitude de raisons, mais les plus courantes sont :

> - Ne pas respecter les principes de base des [Critères de classement](/wiki/Ranking_Criteria).

Reviewer: ```suggestion
- Le non respect des principes de base des [Critères de classement](/wiki/Ranking_Criteria).
```

>   - Contenant des sections mal cadencées

Reviewer: ```suggestion
  - La présence de sections mal cadencées
```

Je dois avouer que je ne comprends pas le terme "cadencées" ici. Si "timing" n'a pas été traduit dans les autres pages, remplace ça par "mal timées" ou "avec un mauvais timing"

>   - Contenant des [objets](/wiki/Hit_object) placés de manière aléatoire sur la grille et/ou la timeline.

Reviewer: ```suggestion
  - La présence [d'objets](/wiki/Hit_object) placés de manière aléatoire sur la grille et/ou la timeline.
```

  - Plans de beatmap extrêmement difficiles ou non conventionnels (par exemple, composés d'un seul [spinner](/wiki/Hit_object/Spinner)).

> - Ne pas respecter le [Code de conduite de modding et de mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping).

Reviewer: ```suggestion
- Le non respect [Code de conduite de modding et de mapping](/wiki/Rules/Code_of_Conduct_for_Modding_and_Mapping).
```
